js_modules/dagster-ui/packages/ui-core/src/asset-graph/useAssetGraphData.tsx

```diff
@@ -44,6 +45,7 @@ export function useAssetGraphData(opsQuery: string, options: AssetGraphFetchScop
       groupSelector: options.groupSelector,
     },
   });
+  useQueryResultDependency(fetchResult, 'ASSET_GRAPH_QUERY');
```

Comment:
this is a no-op since `TraceContext` isn't hooked up to traces yet?

Reply:
Right. It'll only be hooked up in cloud, so in OSS all of this code will no-op